test/unit/org/apache/cassandra/sensors/SensorsReadTest.java

```diff
@@ -258,6 +286,28 @@ private void assertRequestAndRegistrySensorsEquality(Context context)
         assertThat(registrySensor).isEqualTo(localSensor);
     }
 
+    private void assertResponseSensors(double requestValue, double registryValue)
```

Comment:
What about also checking that the `message` can be serialized and deserialized correctly (which would verify the Coordinator will read the correct value)?

Reply:
Makes sense. See https://github.com/datastax/cassandra/pull/950/commits/2b95a0f22029abbccf8a8e0a6168a557c13a2891